src/devices/Ssd1351/Command/Commands.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
copyright header missing (same comment for all files where it applies - see other .cs files for example)